purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesFactory.kt

```diff
@@ -104,6 +98,15 @@ internal class PurchasesFactory(
                 diagnosticsTracker
             )
 
+            val customerInfoResponseHandler = CustomerInfoResponseHandler(billing)
+            val backend = Backend(
+                appConfig,
+                dispatcher,
+                diagnosticsDispatcher,
+                httpClient,
+                backendHelper,
+                customerInfoResponseHandler
```

Comment:
We haven't modified the `Backend` constructor for this so it's failing to compile